project/src/main/editor/puzzle/playfield-editor-control.gd

```diff
@@ -82,7 +82,7 @@ func _on_PlayfieldNav_add_tiles_key_pressed() -> void:
 			break
 	
 	if not new_key:
-		push_error("Couldn't add tiles key: Too many conflicts")
+		push_warning("Couldn't add tiles key: Too many conflicts")
```

Comment:
These feel better suited as errors; you are pressing a button, and the button has no effect

Reply:
Reverted.